components/CustomMenu.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
```

Comment:
This file seems deprecated, we can remove it directly

Reply:
I've removed this file. Thanks for the catch. 😄